scripts/apps/archive/views/preview.html

```diff
@@ -46,13 +46,13 @@
               <span ng-if="item._type === 'archived'" class="stage" tooltip="{{:: 'Archived from' | translate}} {{deskName}} / {{ stage }}" tooltip-placement="bottom" ng-show="stage"><b>{{:: 'Archived from' | translate}} {{deskName}}</b> / {{ stage }}</span>
               <span ng-if="item._type !== 'archived'" class="stage" tooltip="{{deskName}} / {{ stage }}" tooltip-placement="bottom" ng-show="stage"><b>{{deskName}}</b> / {{ stage }}</span>
             </div>
-            <div class="preview-header__flex-row preview-header__flex-row--single-line" ng-if="selected.preview.authors">
-              <span class="inline-label">Authors</span><br />
-              <div class="tag-label" ng-repeat="author in selected.preview.authors">{{author.role}} : {{author.sub_label}}</div>
+            <div class="tag-main" ng-if="selected.preview.authors">
+              <span class="inline-label">{{preview_headings.author}}:</span>
+              <span class="tag-label" ng-repeat="author in selected.preview.authors">{{author.role}}:{{author.sub_label}}</span>
             </div>
-            <div class="preview-header__flex-row preview-header__flex-row--single-line" ng-if="selected.preview.keywords">
-              <span class="inline-label">Keywords</span><br />
-              <div class="tag-label" ng-repeat="keyword in selected.preview.keywords" title="Keywords">{{keyword}}</div>
+            <div class="tag-main" ng-if="selected.preview.keywords">
+              <span class="inline-label"><br/>{{preview_headings.keywords}}:</span>
+              <span class="tag-label" ng-repeat="keyword in selected.preview.keywords" title="Keywords">{{keyword}}</span>
```

Comment:
not sure we need the title, but if so it should be the same as label (translated)